tools/rna_tools/sortmerna/sortmerna.xml

```diff
@@ -59,135 +25,87 @@ sortmerna --version 2>&1|grep 'SortMeRNA version'
     </version_command>
     <command>
 <![CDATA[
-    #set $ref = ''
-    #set $sep=''
-    #if str( $databases_type.databases_selector ) == 'history'
-        #for $db in $databases_type.database_name
-            #set $ref += $sep + str($db) + ',' + $os.path.splitext($os.path.basename(str($db)))[0]
-            #set $sep = ':'
-        #end for
-    #else if str( $databases_type.databases_selector ) == 'cached_to_index'
-        #for $db in $databases_type.input_databases.fields.path.split(",")
-            #set $ref += $sep + $db + ',' + $os.path.splitext($db)[0] + '-reindexed'
-            #set $sep = ':'
-        #end for
-    #else
-        #for $db in $databases_type.input_databases.fields.path.split(",")
-            #set $ref += $sep + $db + ',' + $os.path.splitext($db)[0]
-            #set $sep = ':'
-        #end for
-    #end if
 
-    #if str( $databases_type.databases_selector ) != 'cached'
-        indexdb_rna 
-            --ref '$ref'
+    @PREPROCESSING@
+    
+    mkdir -p './aligned' &&
+
+    sortmerna
+        #for $i, $reference in enumerate($ref)
+            --ref '$reference'
+        #end for
+        #if str( $databases_type.databases_selector ) != 'cached'
```

Comment:
Also I cannot completely. Reference FASTA and indices can be provided in that way ` ./sortmerna --ref ./rRNA_databases/silva-bac-16s-id90.fasta,./index/silva-bac-16s-db`

Perhaps @bebatut could have a look, since she wrote the datamanager. Otherwise I propose to keep this section without changes.